owasp-suppression.xml

```diff
@@ -357,4 +357,12 @@
         <packageUrl regex="true">^pkg:maven/org\.wildfly\.openssl/wildfly\-openssl\-windows\-x86_64@.*$</packageUrl>
         <cve>CVE-2022-1278</cve>
     </suppress>
+    <!--    This cve concerns https://github.com/billdavidson/JSONUtil/issues/10-->
```

Comment:
I would add some info why it's suppressed. Something similar to:
 </suppress> <!-- False positive, H2 web console is used in development process. Not shared in production --> <suppress> <notes><![CDATA[ file name: h2-1.4.197.jar ]]></notes> <packageUrl regex="true">^pkg:maven/com\.h2database/h2@.*$</packageUrl> <cve>CVE-2022-45868</cve> </suppress>